dev-utils/test-servers.js

```diff
@@ -25,8 +25,10 @@
 
 const express = require('express')
 const serveIndex = require('serve-index')
+const proxy = require('express-http-proxy')
 const { join } = require('path')
 const { runIntegrationTest } = require('./integration-test')
+const { DEFAULT_APM_SERVER_URL } = require('./test-config')
```

Comment:
We should use the `getTestEnvironmentVariables` function and get the serverUrl instead of exporting this constant since it can be changed via env.

Reply:
The serverUrl we get from the env variables is used for the tests to connect to, in other words this proxy is creating a server that tests will connect to. But the apm servers actual destination is a different one. We can add another env variable for that but at this point I think that's unnecessary.